libevm/VM.cpp

```diff
@@ -308,13 +372,13 @@ bytesConstRef VM::execImpl(u256& io_gas, ExtVMFace& _ext, OnOpFunc const& _onOp)
 				m_stack.pop_back();
 			}
 
-			unsigned inOff = m_stack.back().convert_to<unsigned>();
+			unsigned inOff = (unsigned)m_stack.back();
 			m_stack.pop_back();
-			unsigned inSize = m_stack.back().convert_to<unsigned>();
+			unsigned inSize = (unsigned)m_stack.back();
 			m_stack.pop_back();
-			unsigned outOff = m_stack.back().convert_to<unsigned>();
```

Comment:
Which are the "these" that you think are regressions?  Git says there are no merge conflicts.  So I probably misunderstand.